src/commands/plugins.rs

```diff
@@ -63,7 +63,6 @@ pub struct Install {
         name = PLUGIN_NAME_OPT,
         conflicts_with = PLUGIN_REMOTE_PLUGIN_MANIFEST_OPT,
         conflicts_with = PLUGIN_LOCAL_PLUGIN_MANIFEST_OPT,
-        required_unless_present_any = [PLUGIN_REMOTE_PLUGIN_MANIFEST_OPT, PLUGIN_LOCAL_PLUGIN_MANIFEST_OPT],
```

Comment:
Not sure if it's intentional that this PR changes the `install` command?  I think you might have intended to remove this from line 170 instead?

Reply:
You’re right, obviously. Awkward mistake.